kwok/charts/crds/karpenter.sh_nodeclaims.yaml

```diff
@@ -454,7 +343,7 @@ spec:
             - spec
           type: object
       served: true
-      storage: true
+      storage: false
```

Comment:
Should we be changing the storage version here if we're only wanting to add the conversion webhook support here

Reply:
Moved to a separate PR. https://github.com/kubernetes-sigs/karpenter/pull/1357